SetupDataPkg/Library/ConfigVariableListLib/UnitTest/ConfigVariableListLibUnitTest.inf

```diff
@@ -0,0 +1,38 @@
+## @file
+# Unit tests of the ConfigVariableListLib instance.
+#
+# Copyright (C) Microsoft Corporation.
```

Comment:
nit:
```suggestion
# Copyright (c) Microsoft Corporation.
```

Reply:
I'll fix up. This was copy/paste from another lib.